controllers/comments.controllers.js

```diff
@@ -18,4 +21,20 @@ const getAllCommentsByArticleId = (req, res, next) => {
     });
 };
 
-module.exports = { getAllCommentsByArticleId };
+const postArticleComment = (req, res, next) => {
+  const { body } = req;
+  const { article_id } = req.params;
+
+  selectArticleById(article_id)
```

Comment:
This is a nifty way to check that the _article_id_ is valid by using your `selectArticleById` function, however...
The table `comments` has a column `article_id` that _references_ the articles table. This means that psql will throw an error if the article_id cannot be found! The error code won't be a 22P02, you will need to handle for a new error code

Try removing `selectArticleById` from this controller once your tests have the correct paths, and see what error you get back from psql 🤔

Reply:
done